jira/resources.py

```diff
@@ -956,13 +956,19 @@ def add_user(
             groups (Optional[Union[str,List,Tuple]]): a group or groups to add to the role
         """
 
+        data = None
         if users is not None and isinstance(users, str):
             users = (users,)
+            data = {"user": users}
+            if groups is not None:
+                raise NotImplementedError("JIRA can only add users XOR groups")
```

Comment:
```suggestion
        if users is not None: 
            if isinstance(users, str):
                users = (users,)
            data = {"user": users}
            if groups is not None:
                raise NotImplementedError("JIRA can only add users XOR groups")
```

This function accepts both strings and a list/tuple already, so we need to be a bit smarter on the handling here